.github/workflows/ci.yml

```diff
@@ -43,7 +43,7 @@ jobs:
     container:
       image: ghcr.io/sparklemotion/nokogiri-test:${{matrix.image}}
     steps:
-      - uses: actions/checkout@v1 # v1 because of https://github.com/actions/checkout/issues/334
```

Comment:
This should use v4 (or really 692973e3d937129bcbf40652eb9f2f61becf3332).

Reply:
Can't use v4 because of the issue linked in the comment.

Reply:
Got it. I briefly checked the issue and it was reported for v2 and there were comments saying it had been fixed, but that seems not to be the case. Odd that it would be broken for so long.

Reply:
I haven't tried it in a few years, maybe it would work if we bumped it. But I got the distinct impression that Github only really cares if the actions runs on their official runners, and won't fully support running in arbitrary containers. Which, TBH, is a completely defensible position to take.